lib/browsers.js

```diff
@@ -6,13 +6,17 @@ exports.chrome = {
   find: function () {
     // Joined with filename (or [browser name].exe)
     this.dir('LOCALAPPDATA', 'Google\\Chrome\\Application')
-
+    this.dir('LOCALAPPDATA', 'Google\\Chrome Beta\\Application')
+    this.dir('LOCALAPPDATA', 'Google\\Chrome Dev\\Application')
     // Chrome Canary
     this.dir('LOCALAPPDATA', 'Google\\Chrome SxS\\Application')
 
     // programFiles() is a shortcut to dir() that checks both
     // "Program Files" and "Program Files (x86)" if on 64-bit Windows
     this.programFiles('Google\\Chrome\\Application')
+    this.programFiles('Google\\Chrome Beta\\Application')
+    this.programFiles('Google\\Chrome Dev\\Application')
+    this.programFiles('Google\\Chrome SxS\\Application')
```

Comment:
Is it still the case that the offline Google Chrome installer (aka the standalone installer) installs to program files, while the normal installer installs to local app data?

Reply:
Nope, I just tried Chrome Dev, it installs to program files too. Oh well.

Reply:
It seems inconsistent, for some reason my Chrome Canary has been installed to local app data, but none of the others. No idea if I used a different installer by accident or if that's specific to canary, but it seems sensible to just check both imo.